src/ILCompiler.ReadyToRun/src/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1665,6 +1673,73 @@ private void expandRawHandleIntrinsic(ref CORINFO_RESOLVED_TOKEN pResolvedToken,
             throw new RequiresRuntimeJitException($"{MethodBeingCompiled} -> {nameof(getMethodSync)}");
         }
 
+        private byte[] _bbCounts;
+        private ProfileDataNode _profileDataNode;
+
+        partial void PublishProfileData()
+        {
+            if (_profileDataNode != null)
+            {
+                CORINFO_METHOD_INFO methodInfo;
+                var methodIL = Get_CORINFO_METHOD_INFO(MethodBeingCompiled, null, &methodInfo);
```

Comment:
`Get_CORINFO_METHOD_INFO` will do a bunch of work and pin a bunch of stuff that we then throw away.

```suggestion
                MethodIL methodIL = _compilation.GetMethodIL(MethodBeingCompiled);
```

Reply:
Thanks